vendor/vendor.json

```diff
@@ -146,6 +146,14 @@
 			"version": "v15.1.0",
 			"versionExact": "v15.1.0"
 		},
```

Comment:
Please delete the old package if there's no any reference in the code base.

Reply:
done

Reply:
Done.